salt/metalk8s/orchestrate/upgrade/init.sls

```diff
@@ -30,6 +30,11 @@ Upgrade etcd cluster:
   {%- set kubeconfig = "/etc/kubernetes/admin.conf" %}
   {%- set context = "kubernetes-admin@kubernetes" %}
 
+Refresh the package repository for {{ node }}:
+  salt.function:
+    - tgt: {{ node }}
+    - name: pkg.refresh_db
+
 Set node {{ node }} version to {{ dest_version }}:
```

Comment:
Just add a require on `salt: Refresh the package repository for {{ node }}` to be cleaner

Reply:
ok